src/renderer/components/back-button/back-button.test.tsx

```diff
@@ -24,7 +24,7 @@ describe('back button', () => {
         >
           <img
             className=""
-            src="renderer/components/back-button/back-button.svg"
+            src="renderer\\components\\back-button\\back-button.svg"
```

Comment:
Don't use Windows slashes. All the builds use Unix systems so this will fail all of the CIs.
The path is extracted in `src/jest-file-transformer.js`. Modify this file to always use backslashes.